src/webgl-context/polyfill-vertex-array-object.js

```diff
@@ -358,7 +358,7 @@ export default function polyfillVertexArrayObject(WebGLRenderingContext, gl) {
       }
 
       if (!this.__OESVertexArrayObject) {
-        console.debug('Setup OES_vertex_array_object polyfill');
+        // console.debug('Setup OES_vertex_array_object polyfill');
```

Comment:
We can just cut this! 

BTW, maybe consider what version of node are you on? console.debug should be available on recent versions.